{
	"id": "CVE-2020-29396",
	"tags": [
		{
			"label": "Product",
			"message": "Odoo Community"
		},
		{
			"label": "Product",
			"message": "Odoo Enterprise"
		},
		{
			"label": "Version",
			"message": "<= 13.0 "
		},
		{
			"label": "Version",
			"message": ">= 11.0 "
		},
		{
			"label": "Vulnerability",
			"message": "CWE-267: Privilege Defined With Unsafe Actions"
		}
	],
	"description": " A sandboxing issue in Odoo Community 11.0 through 13.0 and Odoo Enterprise 11.0 through 13.0, when running with Python 3.6 or later, allows remote authenticated users to execute arbitrary code, leading to privilege escalation.",
	"poc": {
		"references": [
			"https://www.oracle.com/security-alerts/cpujul2022.html"
		],
		"githubs": [
			"https://github.com/ARPSyndicate/cvemon",
			"https://github.com/shaharsigal/Final-Project-Cyber-Security"
		]
	}
}
